bittensor/utils/__init__.py

```diff
@@ -308,20 +308,24 @@ def is_valid_destination_address( address: Union[str, bytes] ) -> bool:
         # Check if ed25519
         if address.startswith('0x'):
             if not is_valid_ed25519_pubkey( address ):
-                bittensor.__console__.print(":cross_mark: [red]Invalid Destination Public Key[/red]: {}".format( address ))
+                if not no_prompt:
```

Comment:
why if not no_prompt? will this potentially fail silently if the user passes `--no_prompt` along with a destination key?

Reply:
it will fail silently, but it will still return False so then it's up to the calling function to give output.